java/memory/memory-core/src/main/java/org/apache/arrow/memory/BaseAllocator.java

```diff
@@ -41,7 +41,7 @@
 abstract class BaseAllocator extends Accountant implements BufferAllocator {
 
   public static final String DEBUG_ALLOCATOR = "arrow.memory.debug.allocator";
-  public static final int DEBUG_LOG_LENGTH = 6;
+  public static final int DEBUG_LOG_LENGTH = 30;
```

Comment:
was this a permanent change or accidentally left in?

Reply:
Good catch, this was leftover from debugging.